doc/cephfs/client-config-ref.rst

```diff
@@ -4,115 +4,115 @@
 
 ``client acl type``
 
-:Description: Set to ``"posix_acl"`` to enable POSIX ACLs.
+:Description: Set the ACL type. Currently, only possible value is ``"posix_acl"`` to enable POSIX ACL, or an empty string.
 :Type: String
-:Default: ``""`` (N/A)
+:Default: ``""`` (empty string)
```

Comment:
The N/A was meant to mean "no ACL enforcement", i.e. just normal POSIX permissions.